packages/sdk-v2/src/sdk/newsfeed/OrbisCachedFeed.ts

```diff
@@ -0,0 +1,123 @@
+import { Orbis } from "@orbisclub/orbis-sdk";
+import { isArray } from "lodash";
+import { createNewsFeedDb } from "./utils";
+import { AsyncStorage } from "../storage";
+import { IpfsStorage, isValidCID } from "../ipfs/sdk";
+import { batch } from "src/utils";
```

Comment:
@johnsmith-gooddollar @sirpy 
Likely a ts configuration that can fix this (but I dont know which one, this has been looked into before)

when sdk-v2 imported as package it is unable to resolve absolute paths.
need to be relative 

fixed in ui pr: https://github.com/GoodDollar/GoodWeb3-Mono/pull/107/commits/3066347e44e292c6d7a2d238f5b86e8a9d31521d